Automattic/woocommerce-payments
# Dismissible message for applepay error

Fixes #4610

#### Changes proposed in this Pull Request
In case of the country error message for Apple Pay, the error message should be dismissible. 
Context: pdjVC1-Yt-p2#comment-1356 









#### Testing instructions
* Set the merchant country other than the ones listed in https://support.apple.com/en-us/HT207957 .
* Enable Apple Pay
* Check if the country error notification is dismissible





-------------------


**Post merge**
